packages/libs/eda/src/lib/core/components/computations/plugins/correlationAssayAssay.tsx

```diff
@@ -182,6 +183,81 @@ export function CorrelationAssayAssayConfiguration(
     });
   }, []);
 
+  // Content for the expandable help section
+  const helpContent = (
+    <div className={cx('-HelpInfoContainer')}>
+      <H6>What is correlation?</H6>
+      <p>
+        The correlation between two variables (taxa, genes, sample metadata,
+        etc.) describes the degree to which their presence in samples
+        co-fluctuate. For example, the Age and Shoe Size of children are
+        correlated since as a child ages, their feet grow.
+      </p>
+      <p>Here we look for correlation between:</p>
+      <ol>
+        <li>Abundance of taxa at a given taxonomic level</li>
+        <li>Abundance of functional data (e.g. pathways, genes)</li>
+      </ol>
+      <br></br>
+      <H6>Inputs:</H6>
+      <p>
+        <ul>
+          <li>
+            <strong>Taxonomic Level.</strong> The taxonomic abundance data to
+            used in the calcualtion
+          </li>
+          <li>
+            <strong>Functional Data.</strong> The pathway, metabolic, or gene
```

Comment:
Maybe we can be more specific here? Pathway what? Is it accurate/ clear to say genes here?

Reply:
Haha I was intentionally trying to leave it vague because we have multiple types of pathway data. I think gene is similarly a good term that can cover lots of data types. I don't want to have to update this list each time we get a slightly different data type.